docs/service/connection-properties.md

```diff
@@ -0,0 +1,16 @@
+# Connection Properties
+
+Following are properties for different types of connections. 
+
+| Property                                                              | Description                                                      |
+| ---------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| **Database**                                                            |                                                                  |
+| username                              | Username to be used in the connection   |
```

Comment:
You put the type in every other description except this one. Maybe split it to its own column or add it here?